src/contrib/ivohra/Data_Analysis/fs_particle_distribution.C

```diff
@@ -0,0 +1,172 @@
+//____________________________________________________________________________
+/*
+\brief    Plot a normalized 2D histogram depicting the distribution of the 
+          number of each type of final state particle produced per event 
+\author   Ishaan Vohra <ishaanklv@gmail.com>
+          Phillips Exeter Academy
+\created  August 16, 2022
+*/
+//____________________________________________________________________________
+
+
+#include <iostream>
+#include <string>
+
+#include "GHepParticle.h"
+#include "ScatteringType.h"
+#include "ProcessInfo.h"
+#include "GHepParticle.h"
+#include "TObjArray.h"
+#include "PDGLibrary.h"
+
+#include "TString.h" 
+#include "TFile.h"
+#include "TTree.h"
+#include "TH2.h"
+#include "TAxis.h"
+
+#include "NtpMCTreeHeader.h" 
+#include "NtpMCEventRecord.h" 
+#include "EventRecord.h" 
+#include "BaryonResUtils.h"
+
+#include "GHepParticle.h"       
+#include "PDGUtils.h"
+#include "GHepStatus.h"
+
+#include "PDGCodes.h"
+#include "TDatabasePDG.h"
+
+
+
+void fs_particle_distribution()
+{
+
+using namespace genie;
+
+// Open the GHEP ROOT file
+
+  string filename = "/hepstore/ivohra/miniboone1.ghep.root";
```

Comment:
I'd suggest making `filename` a parameter of the script's function ala
`void fs_particle_distribution(string filename = "/hepstore/ivohra/miniboone1.ghep.root")`
as it would allow users to use a different file without having to edit the script.